README.md

```diff
@@ -19,6 +19,29 @@ to use:
     you will end up with a TAB file in `target/` that you can program onto your
     Tock board.
 
+## Using libtock-rs
+
+The easiest way to start using libtock-rs is adding an example to the examples folder.
+The boiler plate code you would write is
+```rust
+#![no_std]
+#![feature(alloc)]
+
+extern crate tock;
```

Comment:
`extern crate alloc`?

Reply:
You don't necessarily need this (see the blink example).

Reply:
Agreed. Then, we also do not need `#![feature(alloc)]`. Maybe you could add an explanation like "If you need heap based allocation, just add `#![feature(alloc)]` and `extern crate alloc;`"?